x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -16,21 +17,31 @@ import {
   LATEST_FINDINGS_INDEX_DEFAULT_NS,
   LATEST_FINDINGS_INDEX_PATTERN,
   LATEST_FINDINGS_INDEX_TEMPLATE_NAME,
+  LATEST_VULNERABILITIES_INDEX_DEFAULT_NS,
+  LATEST_VULNERABILITIES_INDEX_PATTERN,
+  LATEST_VULNERABILITIES_INDEX_TEMPLATE_NAME,
+  VULNERABILITIES_INDEX_NAME,
 } from '../../common/constants';
 import { createPipelineIfNotExists } from './create_processor';
 import { benchmarkScoreMapping } from './benchmark_score_mapping';
-import { latestFindingsPipelineIngestConfig, scorePipelineIngestConfig } from './ingest_pipelines';
+import {
+  latestFindingsPipelineIngestConfig,
+  latestVulnerabilitiesPipelineIngestConfig,
+  scorePipelineIngestConfig,
+} from './ingest_pipelines';
 
 // TODO: Add integration tests
 
 export const initializeCspIndices = async (esClient: ElasticsearchClient, logger: Logger) => {
   await Promise.all([
     createPipelineIfNotExists(esClient, scorePipelineIngestConfig, logger),
     createPipelineIfNotExists(esClient, latestFindingsPipelineIngestConfig, logger),
+    createPipelineIfNotExists(esClient, latestVulnerabilitiesPipelineIngestConfig, logger),
```

Comment:
I would suggest to use the pipeline we already have, `CSP_LATEST_FINDINGS_INGEST_TIMESTAMP_PIPELINE `- it is doing the same thing, isn't it?

Reply:
Yes that's correct. The same processors fields are being used.